portal/exercise_diet/templates/exercise_diet/recipe.html

```diff
@@ -1,14 +1,14 @@
-{% trans %}<button type="button" class="btn btn-secondary-top" data-dismiss="modal"><b>close</b><span>X</span></button>{% endtrans %}
+{% trans %}<button type="button" class="btn btn-secondary-top" data-dismiss="modal"><span>X</span></button>{% endtrans %}
```

Comment:
Should `close` be added as a `title` or another attribute for accessibility? If not, don't see a need to translate `X`